src/core/index.ts

```diff
@@ -231,15 +225,15 @@ const dismissApproval = async (
   context: Context,
   reviewId: number,
 ): Promise<void> => {
-  const pullReq = initPullRelatedRequest(context) as Record<string, string>;
-  const req = (pullReq as unknown) as Octokit.RequestOptions &
-    Octokit.PullsDismissReviewParamsDeprecatedNumber;
-  req["review_id"] = reviewId;
-  req.message = composeReviewDismissalMsg();
+  const req = context.repo({
+    "pull_number": context.payload.pull_request.number,
+    "review_id": reviewId,
+    "message": composeReviewDismissalMsg(),
```

Comment:
Codacy found an issue: [The key 'message' is not sorted alphabetically](https://app.codacy.com/manual/tianhaoz/approveman/pullRequest?prid=6106171)